src/pages/login/Login.js

```diff
@@ -1,38 +1,62 @@
-import React from 'react'
+import React , {useState} from 'react'
 import "./login.css"
 import { Link } from 'react-router-dom'
+import { useLogin } from 'context/LoginContext';
 
 export const Login = () => {
+
+    function setGuestCredential(event) {
+      event.preventDefault();
+      setEmail("adarshbalika@gmail.com");
+      setPassword("adarshbalika");
+    }
+
+      const { loginUser } = useLogin();
+
+      const [ email, setEmail ] = useState("");
+      const [ password, setPassword ] = useState("");
+
+      const loginButton = ({event,email, password}) => {
+      event.preventDefault();
```

Comment:
No need to pass email, password and event
As it is already there in the state 
And you can simply call the loginButton function inside onClick function like this
onClick={login button}, it will automatically pass the event parameter